framework/test/unit/modules/dpos/transaction_assets/register_transaction_asset.spec.ts

```diff
@@ -63,22 +63,42 @@ describe('RegisterTransactionAsset', () => {
 	});
 
 	describe('#validate', () => {
+		describe('schema validation', () => {
+			it('should throw error when username is more than 20 characters', () => {
+				const asset: RegisterTransactionAssetContext = { username: 'abchd1563890087376368' };
+
+				const errors = validator.validate(transactionAsset.schema, asset);
+				expect(errors).toHaveLength(1);
+				expect(errors[0].message).toInclude('should NOT be longer than 20 characters');
+			});
+
+			it('should throw error when username empty string', () => {
+				const asset: RegisterTransactionAssetContext = { username: '' };
+
+				const errors = validator.validate(transactionAsset.schema, asset);
+				expect(errors).toHaveLength(1);
+				expect(errors[0].message).toInclude('should NOT be shorter than 1 characters');
+			});
+		});
+
 		it('should not throw error if valid username is provided', () => {
```

Comment:
its bit confusing.. it says not throw.. but expectation is different

Reply:
Its expecting that it does not throw error with `.not.toThrow`